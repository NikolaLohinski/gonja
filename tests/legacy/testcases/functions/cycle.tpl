{% set row_class = cycler('odd', 'even') -%}
<ul class="browser">
{%- for num in [1, 2, 3] %}
  <li class="{{ row_class.next() }}">{{ num }}</li>
{%- endfor %}
{%- for num in simple.fixed_item_list %}
  <li class="{{ row_class.next() }}">{{ num }}</li>
{%- endfor %}
</ul>
