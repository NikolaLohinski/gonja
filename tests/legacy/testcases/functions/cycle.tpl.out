<ul class="browser">
  <li class="odd">1</li>
  <li class="even">2</li>
  <li class="odd">3</li>
  <li class="even">1</li>
  <li class="odd">2</li>
  <li class="even">3</li>
  <li class="odd">4</li>
</ul>
